src/contacts.cs

```diff
@@ -441,6 +441,794 @@ interface CNLabelContactRelationKey {
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationNephew")]
+		NSString Nephew { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationNephewSistersSon")]
+		NSString NephewSistersSon { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationNephewBrothersSon")]
+		NSString NephewBrothersSon { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationNephewBrothersSonOrHusbandsSiblingsSon")]
+		NSString NephewBrothersSonOrHusbandsSiblingsSon { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationNephewSistersSonOrWifesSiblingsSon")]
+		NSString NephewSistersSonOrWifesSiblingsSon { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationGrandniece")]
+		NSString Grandniece { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationGrandnieceSistersGranddaughter")]
+		NSString GrandnieceSistersGranddaughter { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationGrandnieceBrothersGranddaughter")]
+		NSString GrandnieceBrothersGranddaughter { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationGrandnephew")]
+		NSString Grandnephew { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationGrandnephewSistersGrandson")]
+		NSString GrandnephewSistersGrandson { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationGrandnephewBrothersGrandson")]
+		NSString GrandnephewBrothersGrandson { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepparent")]
+		NSString Stepparent { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepfather")]
+		NSString Stepfather { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepmother")]
+		NSString Stepmother { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepchild")]
+		NSString Stepchild { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepson")]
+		NSString Stepson { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepdaughter")]
+		NSString Stepdaughter { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepbrother")]
+		NSString Stepbrother { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationStepsister")]
+		NSString Stepsister { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationMotherInLawOrStepmother")]
+		NSString MotherInLawOrStepmother { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationFatherInLawOrStepfather")]
+		NSString FatherInLawOrStepfather { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationDaughterInLawOrStepdaughter")]
+		NSString DaughterInLawOrStepdaughter { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationSonInLawOrStepson")]
+		NSString SonInLawOrStepson { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationCousinOrSiblingsChild")]
+		NSString CousinOrSiblingsChild { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationNieceOrCousin")]
+		NSString NieceOrCousin { get; }
+
+		[iOS (13, 0)]
+		[Field ("CNLabelContactRelationNephewOrCousin")]
+		NSString CNephewOrCousin { get; }
```

Comment:
`CNephewOrCousin` => `NephewOrCousin`

Reply:
it's marked as resolved - but it's still has the `C` prefix
please don't resolve discussions before the PR is fixed (it's too easy to forget those items)

Reply:
`C` removed in PR 🙂